docs/src/snap/reference/annotations.md

```diff
@@ -8,6 +8,28 @@ the bootstrap configuration.
 |---------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------|
 | `k8sd/v1alpha/lifecycle/skip-cleanup-kubernetes-node-on-remove` | If set, only microcluster and file cleanup are performed.  This is helpful when an external controller (e.g., CAPI) manages the Kubernetes node lifecycle. By default,  k8sd will remove the Kubernetes node when it is removed from the cluster. | "true"\|"false" |
 | `k8sd/v1alpha/lifecycle/skip-stop-services-on-remove` | If set, the k8s services will not be stopped on the leaving node when removing the node. This is helpful when an external controller (e.g., CAPI) manages the Kubernetes node lifecycle. By default, all services are stopped on leaving nodes. | "true"\|"false" |
+| `k8sd/v1alpha1/csrsigning/auto-approve` | If set, certificate signing requests created by worker nodes are auto approved. | "true"\|"false" |
```

Comment:
Maybe @evilnick / @nhennigan can we change the width of the Description column it renders quite oddly:
<img width="947" alt="image" src="https://github.com/user-attachments/assets/4b6fc99e-3517-444a-a5e3-a3cc21350d44">

Reply:
Will take a look but maybe a table isn't the best option here